WalletWasabi/Hwi/Parsers/HwiParser.cs

```diff
@@ -370,7 +379,7 @@ public static string ToArgumentString(Network network, IEnumerable<HwiOption> op
 			{
 				argumentBuilder.Append(' ');
 			}
-			argumentBuilder.Append(command.ToString().ToLowerInvariant());
+			argumentBuilder.Append(command.Value.ToString().ToLowerInvariant());
```

Comment:
There is `if (command is { })` why does the compiler complains about `command.ToString()`?

Reply:
"Dereference of possible null reference."

Reply:
But how come? `command` is not null there, we are checking that a few lines above.

Reply:
`command` is an `enum` and if you make an enum nullable, then it behaves as `Nullable<HwiCommand>`. 

if `command` were a `class`, it would not report a warning as you wrote.